R/utils.R

```diff
@@ -73,6 +73,14 @@ write_if_different <- function(path, contents, check = TRUE) {
     return(FALSE)
   }
 
+  if (has_windows_le(path)) {
+    line_ending <- "\r\n"
+  } else {
+    line_ending <- "\n"
+  }
+
+  contents <- paste0(paste0(contents, collapse = line_ending), line_ending)
+  contents <- gsub("\r?\n", line_ending, contents)
```

Comment:
```suggestion
  contents <- enc2utf8(gsub("\r?\n", line_ending, contents))
```

and remove the conversion below?

Reply:
Right, good catch, fixed now.